data/kahet/kahet missions.txt

```diff
@@ -738,3 +738,1128 @@ event "pacified ka'sei"
+			`	"Because of that, before we can start our mission, you are going to need to make a stopover at <stopovers> - I was able to convince the prefects on Viminal to reallocate one of the outposts that they had just delivered there to our mission, given it has higher priority status compared to the mining expeditions they are setting up."`
+			`	You help Dusk transport the chosen instruments to your ship; you also make sure to leave enough space for the facilities you're going to take from <stopovers>. As you're walking back toward the ship's controls, Dusk reminds you to bring as many ramscoops with you as possible - you're going to need them in order to complete the mission plan while cloaked.`
+				accept
+	on stopover
+		dialog
+			`Dusk's coordinates indicate the border of a large canyon near the equator of <stopovers>. Unlike most cracks on the surface, this one shows signs of geological activity: you only narrowly avoid a stream ejecting from a geyser in the canyon as you pass overhead. As you land, a row of Remnant structures are clearly visible running parallel to the cliff.`
+			`	The local crew assist in loading the modular laboratory split across three pallets; with them arrives a separate package, containing a surveillance satellite to be deployed near Patir to scan the Ka'sei from orbit.`
+	on abort
+		# Do nothing.
+	on fail
+		dialog
+			`As soon as you approach the <planet>, you see that the Ka'sei are following your ship. You won't be able to enter the settlement safely anymore.`
+
+	npc save
+		government "Test Dummy"
+		personality unconstrained heroic nemesis
+		system "Patir"
+		fleet
+			variant
+				"Cloak Check" 10
+
+mission "Ka'het: Patir Mystery 8B"
+	invisible
+	source "Babiali"
+	destination "Builder Settlement"
+	to offer
+		has "Ka'het: Patir Mystery 8A: offered"
+	on enter Patir
+		dialog
+			`The probe slowly moves away from your ship, opening its external panels and the instrumentation; as you watch it turn into a dim point of light, the Remnant team confirms that it is functioning correctly. Time to land on the asteroid.`
+
+
+mission "Ka'het: Patir Mystery 9"
+	landing
+	name "Explore the neighborhood"
+	description "Visit every system around <system> you can reach, then return to Dusk in <system>."
+	source "Builder Settlement"
+	waypoint "Athiri"
+	waypoint "Chanai"
+	waypoint "Ghila"
+	waypoint "Lathia"
+	waypoint "Maithi"
+	waypoint "Mitera"
+	waypoint "Sarifa"
+	waypoint "Thepa"
+	to offer
+		has "Ka'het: Patir Mystery 8A: done"
+	on offer
+		event "Patir moves"
```

Comment:
You can do `event "Patir moves" 0` to cause this to run instantly. Right now, if the player pulls up the map before they depart, they'll still appear to be in the Ember Waste. You could probably use an `action` node to call the event at the moment that the system moves so that opening up the map before and after that point in the conversation is when the move occurs.